src/scancode/cli.py

```diff
@@ -279,6 +282,9 @@ def scancode(ctx,
 
     scans_cache_class = get_scans_cache_class()
 
+    # Convert ignore (user defined ignored files list) to a dict
+    user_ignore = dict((patt, 'User ignore: Supplied by --ignore') for patt in ignore)
```

Comment:
'User ignore: Supplied by --ignore' is returned when [debugging](https://github.com/nexB/scancode-toolkit/blob/develop/src/commoncode/fileset.py#L34) is turned on, I took a reference from [these descriptions](https://github.com/nexB/scancode-toolkit/blob/develop/src/commoncode/ignore.py#L95), I know it's not the best description to go with, perhaps you could suggest something better?
About dict comprehensions, that really makes more sense, (I actually didn't know about them), thanks for the suggestion 👍